test/WebJobs.Extensions.CosmosDB.Tests/Trigger/CosmosDBListenerTests.cs

```diff
@@ -95,7 +96,7 @@ public async Task GetMetrics_ReturnsExpectedResult()
                 .Setup(m => m.ReadNextAsync(It.IsAny<CancellationToken>()))
                 .Returns(Task.FromResult(response.Object));
 
-            var metrics = await _listener.GetMetricsAsync();
+            var metrics = await ((CosmosDBScaleMonitor)_listener.GetMonitor()).GetMetricsAsync();
```

Comment:
Should we create a new test for `CosmosDBMretricsProvider`? GetMetrics is not part in `CosmosDbListener `anymore.

Reply:
Yeah I think we should create a new class for scale monitor

Reply:
Ported over the tests to CosmosDBScaleMonitorTests